library/src/options.cxx

```diff
@@ -106,57 +106,79 @@ class options::internals
 options::options()
   : Internals(new options::internals)
 {
-  // Loader/Loading
-  this->Internals->init("animation-index", 0);
-  this->Internals->init("camera-index", -1);
-  this->Internals->init("color", std::vector<double>{ 1., 1., 1. });
-  this->Internals->init("emissive-factor", std::vector<double>{ 1., 1., 1. });
-  this->Internals->init("geometry-only", false);
-  this->Internals->init("line-width", 1.0);
-  this->Internals->init("metallic", 0.0);
-  this->Internals->init("normal-scale", 1.0);
-  this->Internals->init("opacity", 1.0);
-  this->Internals->init("point-size", 10.0);
-  this->Internals->init("progress", false);
-  this->Internals->init("roughness", 0.3);
-  this->Internals->init("texture-base-color", std::string());
-  this->Internals->init("texture-emissive", std::string());
-  this->Internals->init("texture-material", std::string());
-  this->Internals->init("texture-normal", std::string());
-  this->Internals->init("up", std::string("+Y"));
-
-  // Rendering/Dynamic
-  this->Internals->init("axis", false);
-  this->Internals->init("background-color", std::vector<double>{ 0.2, 0.2, 0.2 });
-  this->Internals->init("bar", false);
-  this->Internals->init("blur-background", false);
-  this->Internals->init("cells", false);
-  this->Internals->init("colormap",
+  // Loader
+  this->Internals->init("loader.animation.index", 0);
+  this->Internals->init("loader.camera.index", -1);
+  this->Internals->init("loader.up-direction", std::string("+Y"));
+
+  this->Internals->init("loader.geometry-only", false);
+  this->Internals->init("loader.show-progress", false);
+
+  this->Internals->init("loader.default.line-width", 1.0);
+  this->Internals->init("loader.default.point-size", 10.0);
+
+  this->Internals->init("loader.default.color.rgb", std::vector<double>{ 1., 1., 1. });
+  this->Internals->init("loader.default.color.opacity", 1.0);
+  this->Internals->init("loader.default.color.texture", std::string());
+
+  this->Internals->init("loader.default.emissive.factor", std::vector<double>{ 1., 1., 1. });
+  this->Internals->init("loader.default.emissive.texture", std::string());
+
+  this->Internals->init("loader.default.normal.texture", std::string());
+  this->Internals->init("loader.default.normal.scale", 1.0);
+
+  this->Internals->init("loader.default.material.metallic", 0.0);
+  this->Internals->init("loader.default.material.roughness", 0.3);
+  this->Internals->init("loader.default.material.texture", std::string());
```

Comment:
I'd prefer `loader.material.xxxx` instead of `loader.default.xxx`.  
And for the last ones:
- `loader.default.material.metallic` -> `loader.material.pbr.metallic`
- `loader.default.material.roughness` -> `loader.material.pbr.roughness`
- `loader.default.material.texture` -> `loader.material.pbr.texture`

Reply:
how do user know at first glance that an option is specific to the default scene then ?

Reply:
and why adding pbr ?
do you plan having non-pbr materials ?

Reply:
From the documentation, the user knows that using the default scene means the material is not loaded.

Reply:
this may not be enough imo